styles/_header.scss

```diff
@@ -1,17 +1,47 @@
 /***** Header *****/
-$header-height: 71px;
+
 
 .header {
-  @include max-width-container;
-  position: relative;
-  align-items: center;
-  display: flex;
-  height: $header-height;
-  justify-content: space-between;
+  background-color: $white;
+  margin: 0 auto;
+  max-width: $max-width-page-container;
+
+  &__inner {
+    @include max-width-container;
+    position: relative;
+    align-items: center;
+    display: flex;
+    @include fluid-value( height, $min-width, $max-width, 100px, 138px);
```

Comment:
what are 100px and 138px? where do these numbers come from?

Reply:
They are numbers that control the height of the logo.